types/should/index.d.ts

```diff
@@ -88,10 +88,12 @@ interface ShouldAssertion {
   keys(...allKeys: string[]): ShouldAssertion;
   keys(allKeys: string[]): ShouldAssertion;
   enumerable(property: string, value?: any): ShouldAssertion;
+  size(size: number): ShouldAssertion;
+  value(key: string | number, value: any): ShouldAssertion;
   enumerables(...properties: string[]): ShouldAssertion;
   startWith(expected: string, message?: any): ShouldAssertion;
   endWith(expected: string, message?: any): ShouldAssertion;
-  throw(message?: any): ShouldAssertion;
+  throw(message?: any, properties?: Object): ShouldAssertion;
```

Comment:
@DanielRosenwasser Hm, interesting. I was just opting for `Object` because it's used elsewhere in the file already. Sounds like changing all to `object` might be a good idea anyway 👍